src/test/kotlin/no/digdir/fdk/searchservice/integration/FilterTest.kt

```diff
@@ -393,5 +394,38 @@ class FilterTest: ApiTestContext() {
             Assertions.assertEquals(0, result.size)
         }
     }
-}
 
+    @Nested
+    inner class Relations {
+        @Test
+        fun `get relations to dataset`() {
+            val searchBody = mapper.writeValueAsString(SearchOperation(filters = SEARCH_FILTER.copy(
+                relations = SearchFilter(TEST_DATASET_FILTERS.uri))
+            ))
+
+            val response = requestApi(DATASETS_PATH, port, searchBody, HttpMethod.POST)
+            Assertions.assertEquals(200, response["status"])
+
+            val result: List<SearchObject> = mapper.readValue(response["body"] as String)
+            Assertions.assertNotEquals(0, result.size)
+
+            result.forEach { searchObject ->
+                searchObject.relations?.forEach { relation ->
+                    Assertions.assertTrue(relation.uri == TEST_DATASET_FILTERS.uri)
```

Comment:
ikke 100% på at dette fungerer, men noe ala dette
```suggestion
                Assertions.assertTrue(searchObject.relations?.map { it.uri }.any { it.uri == TEST_DATASET_FILTERS.uri)
```

Reply:
Så ikke denne, så prøvde meg på dette:

```
val validValues = result.all { searchObject ->
                searchObject.relations?.any { relation ->
                    relation.uri == TEST_DATASET_FILTERS.uri
                } ?: false
            }
```

Reply:
👍 skal funke det